build.gradle.kts

```diff
@@ -343,26 +362,22 @@ tasks.shadowJar {
 		from(zipTree(remapJar.archiveFile.get())) {
 			duplicatesStrategy = DuplicatesStrategy.EXCLUDE
 			exclude("fabric.mod.json", "mcmod.info", "META-INF/mods.toml", "pack.mcmeta")
+
```

Comment:
This will almost definitely remove the `@Mod` from the LexForge entrypoint. That's why I explicitly listed the target classes.

Reply:
Please restore the old whitelist approach. I don't see why a blacklist approach is an improvement.

Reply:
I would say that it's an improvement because it allows more lexforge entrypoints to be added in this manner without explicitly adding them to the whitelist. Correct me if I'm wrong, but I don't think that anything will need to be added to the blacklist. The only minor downside here is that it takes slightly longer to read and write every class file, but that shouldn't be a problem.

Reply:
I prefer the whitelist approach. Please revert back to it.